Moodis/Moodis/Extensions/String.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Moodis.Extensions
+{
+    public static class String
```

Comment:
Change class name because i already see someone using String and not recognizing that they may be calling a non system class.

Reply:
Good point